ipatests/prci_definitions/temp_commit.yaml

```diff
@@ -72,7 +72,7 @@ jobs:
       class: RunPytest
       args:
         build_url: '{fedora-latest/build_url}'
-        test_suite: test_integration/test_REPLACEME.py
+        test_suite: test_integration/test_ipahealthcheck.py::TestIpaHealthCheck
         template: *ci-master-latest
         timeout: 3600
```

Comment:
You need to increase the timeout to 6300, similar to what is used in nightly_latest https://github.com/freeipa/freeipa/blob/c2aae876f04c127b7b2eb6dad8677a3ae8ceefb8/ipatests/prci_definitions/nightly_latest.yaml#L1409-L1419